src/django/api/migrations/0007_add_default_state.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.13 on 2022-09-29 22:41
+
+import os
+import json
+from django.conf import settings
+from django.db import migrations
+from django.contrib.gis.geos import GEOSGeometry, MultiPolygon
+
+def add_default_state_nc(apps, schema_editor):
+    # Utility relies on a many-to-one relationship with State
+    # We should populate db with a State so we have ability to
+    # add upcoming state field on Utility with default FK id
+    State = apps.get_model('api', 'State')
+    with open(os.path.join(settings.BASE_DIR, 'data/NC_BOUNDARY.geojson'), 'r') as f:
+        NC_GEOJSON = json.load(f)
+    NC_POLYGON = GEOSGeometry(str(NC_GEOJSON['features'][0]['geometry']))
+    # Turn Polygon into type MultiPolygon to match shape field
+    NC_MULTIPOLYGON = MultiPolygon([NC_POLYGON])
+    NC_STATE = State(id="NC", name="North Carolina", shape=NC_MULTIPOLYGON)
+    NC_STATE.save()
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0006_create_state'),
+    ]
+
+    operations = [
+        migrations.RunPython(add_default_state_nc),
```

Comment:
Another team discussion topic, but I try to always add a reverse operation for `RunPython`--at the very least [RunPython.noop](https://docs.djangoproject.com/en/3.2/ref/migration-operations/#django.db.migrations.operations.RunPython.noop)--because it makes unapplying migrations easier (which does come up in development, when switching branches).

Reply:
Would second a team discussion on this one! I popped `RunPython.noop` in for myself when I was reverting some of these migrations for edits but noticed some of our other projects didn't have one so I took it out....but maybe worth bringing back in regardless for our own ease of use?